.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "windows-machine-config-bootstrapper"]
 	path = windows-machine-config-bootstrapper
 	url = https://github.com/openshift/windows-machine-config-bootstrapper
-	branch = release-4.8
+	branch = community-4.8
```

Comment:
Was this generated using hack/update_submodules.sh? Otherwise, should be called out in the commit msg.

Reply:
Yes that was generated